src/constants/testingConstant.js

```diff
@@ -0,0 +1,9 @@
+const testParams = {
+    name: 'sangmean',
+    phoneNum: 7777,
+    address: { City: 'Calgary', Country: 'Canada', street: '57Ave', postalCode: 'T2H' },
+};
+
+module.exports = {
+    testParams,
+};
```

Comment:
I think this should live in each specific test file which uses these values.

Changing one of these values will affect the entire test.

Reply:
Ok!! I will fix it.